tests/ui/target_modifiers/defaults_check.rs

```diff
@@ -0,0 +1,20 @@
+// Tests that default unspecified target modifier value in dependency crate is ok linked
+// with the same value, explicitly specified
+//@ aux-crate:default_reg_struct_return=default_reg_struct_return.rs
+//@ compile-flags: --target i686-unknown-linux-gnu -Cpanic=abort
+//@ revisions:error ok ok_explicit
+//@[ok] compile-flags:
+//@[ok_explicit] compile-flags: -Zreg-struct-return=false
+//@[error] compile-flags: -Zreg-struct-return=true
+//@ needs-llvm-components: x86
+//@[ok] build-pass
+//@[ok_explicit] build-pass
+
+#![crate_type = "lib"]
+//[error]~^ ERROR 13:1: 13:1: mixing `-Zreg-struct-return` will cause an ABI mismatch
```

Comment:
Suggestion: drop the exact line numbers, the error message itself is what we care about for this error annotation.

```suggestion
//[error]~^ ERROR mixing `-Zreg-struct-return` will cause an ABI mismatch
```

Reply:
Fixed.